### PR TITLE
rails api quickstart

### PR DESCRIPTION
**Pull request by mooreds**
Todo

* [ ] Archive old rails API example
* [x] Update rails api blog post to point to this.
* [x] Add to nav
* [ ] Should we show refresh grant - not at this time
* [ ] Should we show refresh token revocation - not at this time

### CODE REVIEW

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+description: Integrate your Ruby on Rails API with FusionAuth
+navcategory: getting-started
+prerequisites: Ruby, bundler and Rails
+technology: Ruby on Rails
+language: Ruby
+---
+
+## Integrate Your {{page.technology}} API With FusionAuth
+
+{% include docs/integration/_intro-api.md %}
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:13]: These links appear to be broken when I try to run this locally. Are we missing some pages?

Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:13]: Ooops, I need to commit and push the file up.

Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:13]: Should be there now.

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+```shell
+touch Gemfile
+```
+
+Now, copy and paste the following file into `Gemfile`.
+
+```ruby
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/ruby/Gemfile %}
+
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:47]: need closing backticks here

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+Now you are going to create a {{page.technology}} API. While this section builds a simple {{page.technology}} API, you can use the same configuration to build a more complex {{page.technology}} API.
+
+First, create the skeleton of the {{page.technology}} API. Rails has a nice generator to build this out.
+
+```shell
+rails new myapi --api && cd myapi
+```
+
+Now, create a new `Gemfile` file.
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:88]: we did provide explicit `touch` and `copy` instructions for this earlier. Rails already creates this one. This file also may bump with the ruby version (I was running 2.7.5 running through this but the included one references 3.2.2. Looks like we're just adding deps.

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+ruby setup.rb
+```
+
+If you want, you can http://localhost:9011[login to your instance] and examine the new API configuration the script created for you.
```
Comment (vcampitelli) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:76]:
```suggestion
If you want, you can [log into your instance](http://localhost:9011) and examine the new API configuration the script created for you.
```

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+This tells {{page.technology}} to return the content generated by the `messages_controller.rb` file when the `/messages` path is set up.
+
+You can now start up your server. You should do it in a new terminal window so that you can continue to edit the {{page.technology}} code.
+
+```shell
+bundle e rails s -p 4001
+```
+
+And visit [http://localhost:4001/messages](http://localhost:4001/messages) and view the JSON.
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:128]: This renders an error on line 7 of include.
Screenshot 2023-05-17 at 1 18 29 PM 1

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+Now, back to the terminal where your server is running. Stop it (using `control-C`) and restart it.
+
+```shell
+bundle e rails s -p 4001
+```
+
+Visit [http://localhost:4001/messages](http://localhost:4001/messages), you'll get an error:
+
+```json
+{"error":"Invalid JWT token : Decode Error"}
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:153]: I got `{"error":"Missing token cookie and Authorization header"}`

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+## Testing the API Flow
+
+There are a number of ways to get an access token, as mentioned, but for clarity, let's use the login API to mimic a client. 
+
+Run this command in a terminal:
+
+```shell
+curl -H 'Authorization: YOUR_API_KEY_FROM_ABOVE' \
+     -H 'Content-type: application/json' \
+     -d '{"loginId": "admin@example.com", "password":"password","applicationId": "e9fdb985-9173-4e01-9d73-ac2d60d1dc8e"} \
```
Comment (lyleschemmerling) [inline site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:167]: We should call out that this should be an existing user. `admin@example.com` won't exist here and likely the only user that has been set up is the admin user created at the start.

also 
```suggestion
     -d '{"loginId": "admin@example.com", "password":"password","applicationId": "e9fdb985-9173-4e01-9d73-ac2d60d1dc8e"}' \
```

**Review by lyleschemmerling — APPROVED** [review]
left notes as I ran though it, but the one error aside I got through it all

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md (hunk 7 above)
Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:167]: Great catch.

File: site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md (hunk 5 above)
Comment (mooreds) [reply site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md:132]: Thanks, resolved.

**Pull request merged**